tests/Modules/MotdModule.Tests/HttpServiceTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using EvoSC.Modules.Official.MotdModule.Models;
+using EvoSC.Modules.Official.MotdModule.Services;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace MotdModule.Tests;
+
+public class HttpServiceTests
+{
+    private const string TestServerUri = "https://directus.evoesports.de/items/motd?filter[server][_eq]=testserver";
+    private HttpService _service = new(new Logger<HttpService>(new LoggerFactory()));
+
+    [Theory]
+    [InlineData(TestServerUri)]
+    [InlineData("https://www.google.de")]
+    async Task GetAsyncTest(string uri)
```

Comment:
```suggestion
    async Task Get_Async_Test(string uri)
```

For consistency reasons, we want to use snake case for test names.